mtp_cashbook/misc_views.py

```diff
@@ -121,3 +121,17 @@ def get_template_names(self):
             return ['policy-change-warning.html']
         else:
             return ['policy-change-info.html']
+
+
+class FAQView(TemplateView):
+    title = _('What do you need help with?')
+    template_name = 'faq.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context['contact_us_url'] = reverse_lazy('submit_ticket')
```

Comment:
as this set of urls is not dynamically changed by the view, i think it's easier just to put them into the template and have this as a really dumb view. probs as `{% url ??? as link %}` above each `blocktrans`. that way, if you're looking at the template, you can see where links point to.

Reply:
Ah OK, that was my first implementation, I only moved because it was used twice and I thought it would be nice to avoid the repetition. But I can move back, no problem at all ☺️